draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -103,49 +96,64 @@ This specification defines the format of the Client Attestation that a Client In
 # Terminology
 
 Client Attestation JWT:
-:  A JSON Web Token (JWT) generated by the client backend which is bound to a key managed by a client instance which can then be used by the instance for client authentication.
+:  A JSON Web Token (JWT) generated by the client backend which is bound to a key managed by a Client Instance which can then be used by the instance for client authentication.
 
 Client Attestation Proof of Possession (PoP) JWT:
-:  A Proof of Possession generated by the client instance using the key that the Client Attestation JWT is bound to.
+:  A Proof of Possession generated by the Client Instance using the key that the Client Attestation JWT is bound to.
+
+Client Instance:
+: A deployed instance of a piece of client software.
```

Comment:
```suggestion
: A specific deployment or instantiation of a client application in the OAuth 2.0 framework, representing a single, distinct version of the client software running on a device or in an environment. Each Client Instance can have its own unique configuration, state, and identity, distinguishing it from other deployments of the same client application. This concept is particularly relevant in contexts where a client application might be installed and operated across multiple devices or environments, each requiring individual authentication and authorization processes with OAuth 2.0 Authorization Servers or Resource Servers.
```

Reply:
The existing definition is from an already published RFC and while it is short I believe it accurately captures the required detail